src/main/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationToken.kt

```diff
@@ -0,0 +1,35 @@
+package de.bund.digitalservice.useid.apikeys
+
+import org.springframework.security.core.Authentication
+import org.springframework.security.core.GrantedAuthority
+
+class ApiKeyAuthenticationToken(private val apiKey: String, private var authenticated: Boolean = false) : Authentication {
```

Comment:
Didn't you want to add a URL (static refresh address) to this token? Or is that for another PR?

Reply:
Now the refresh address is in the apiKeys-List, but here still missing? Or is it accessed differently?

Reply:
I'm uncertain. Either we store it there (and return it via `getDetails`) or we look it up in the `apiProperties` when it's needed (i.e. when the `tcToken` is created). What do you think?

Reply:
Hmm.. When looking it up in the `apiProperties` later, we'd have to use the `apiKey` as key, right? I think, I'd rather have it here and getting it via `getDetails`. Then we only have one class, which fetches values from `apiProperties` and the details about the authenticated "user" (which is for now only this URL) are all in this token.

Reply:
I will put it here. 
This class is not the only one fetching data from the `apiProperties` though. The `ApiKeyAuthenticationManager` does that too. But storing all "user" information in one place convinces me.

Reply:
Done, please check again.